Bernard/FavoriteButton.swift

```diff
@@ -58,6 +58,16 @@ import UIKit
         }
     }
     
+    override var isSelected: Bool {
+        willSet {
+            if newValue == true {
+                imageShape.fillColor = imageColorOn.cgColor
+            } else {
+                imageShape.fillColor = imageColorOff.cgColor
+            }
```

Comment:
Not a fix but also `imageShape.fillColor = (newValue) ? imageColorOn.cgColor : imageColorOff.cgColor`